libsplinter/src/events/reactor.rs

```diff
@@ -17,8 +17,9 @@ use std::sync::{
     Arc,
 };
 use std::thread;
+use std::time::Duration;
```

Comment:
typo in commit message: interupts -> interrupts

Reply:
Fixed